### ISSUE TITLE
Map colors are different to the legend colors

### ISSUE THREAD

**Issue by kev007** [kev007/ARTFUL#77]
![image](https://cloud.githubusercontent.com/assets/3058438/23351866/33fb3fd8-fcc4-11e6-93c9-9be57c12700a.png)
Map colors are different to the legend colors. See Germany and 5,700,000 
Most probably due to style fillOpacity: 0.7

https://github.com/kev007/ARTFUL/blob/dfa71c14cfa2b31345a99e195b0a124e96d7d222/server/src/main/webapp/resources/js/app.js#L565-L574

**Comment by pnancke** [comment]
So shouldn't it be enough if we add something like fillOpacity: 0.7 for the legend as well?

**Comment by kev007** [comment]
Yes, but the legend background color is not the same as the map background color.
Was hoping for an easy way to mathematically do (hex value)*((map background)*0.7))

**Comment by kev007** [comment]
changing to wontfix since the legend scale is human-readable
if the change is to be implemented, see: https://www.viget.com/articles/equating-color-and-transparency
